Sources/VaporAWSLambdaRuntime/APIGatewayV2.swift

```diff
@@ -58,11 +58,30 @@ extension Vapor.Request {
         req.headers.forEach { key, value in
             nioHeaders.add(name: key, value: value)
         }
+		
+		if let cookies = req.cookies, cookies.count > 0 {
+			var cookiesStr = ""
+			cookies.enumerated().forEach { entry in
+				
+				if entry.offset > 0 {
+					cookiesStr += "; "
+				}
+				
+				cookiesStr += entry.element
+			}
+			
+			nioHeaders.add(name: "Cookie", value: cookiesStr)
+		}
```

Comment:
```suggestion
        if let cookies = req.cookies, cookies.count > 0 {
            let cookiesString = cookies.joined(separator: "; ")
            nioHeaders.add(name: "cookie", value: cookiesString)
        }
```

@fcobie what do you think about using the `joined(separator: )` function here? Further http headers are case insensitive, but for http2 reasons I lowercased the `cookie` header already.

Reply:
I will run the swift format and I can fix the cookie code. 

I will see if I can figure out how to do a test, but I have very limited time right now. I just wanted to give back the fixes in case anyone else was trying to use the library for Vapor

Reply:
@fcobia

> I will see if I can figure out how to do a test, but I have very limited time right now. I just wanted to give back the fixes in case anyone else was trying to use the library for Vapor

No worries. I'll do it in a separate pr. Thanks for giving back the fixes.